monai/transforms/intensity/array.py

```
@@ -140,6 +141,110 @@ def __call__(self, img: NdarrayOrTensor, mean: float | None = None, randomize: b
+            _mean = ensure_tuple_rep(self.mean, len(img))
+            _std = ensure_tuple_rep(self.std, len(img))
+            for i, d in enumerate(img):
+                img[i] = self._add_noise(
+                    d,
+                    mean=_mean[i],
+                    std=_std[i] * d.std() if self.relative else _std[i],
+                    k=self.degrees_of_freedom,
+                )
+        else:
+            if not isinstance(self.mean, (int, float)):
+                raise RuntimeError(f"If channel_wise is False, mean must be a float or int, got {type(self.mean)}.")
```

Ruff (0.14.3)

Avoid specifying long messages outside the exception class

(TRY003)